i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+        - |-
+          【이형출현】
+          올풀림에서 사람을 습격하는 것만을 생각하는 이형이 나타나 공포에 떠는 사람을 공격한다. GM이 원한다면 이형의 무리 (IZ 237p)x1D6체와 전투를 해도 좋다. 또한 GM은 상황에 따라 다른 에너미를 선택해도 좋다.
+        - |-
+          【광맥출현】
+          있을수 없을 정도로 풍부한 광맥이 나타난다. 황금, 보석, 바다라면 산호 등이다. 사람들이 안색을 바꾸며 광맥으로 몰려든다. 그로 인해 발생하는 혼란은 GM이 결정할 것.
+        - |-
+          【허무출현】
+          올풀림 속에서 허무가 새어나와 약한 이들을 끌고들어간다. 씬에 등장하고 있는 엑스트라는 벌레 한 마리까지 소멸하고 두 번 다시 나타나지 않는다.
+        - |-
+          【오염세계】
+          물은 독을 품고, 대기는 유독성 안개로 가득하다. 등장하고 있는 모든 캐릭터는 【체력】난이도 10의 판정을 해서 실패할 경우 [중독] 5를 받는다. 엑스트라가 어떻게 될지는 GM이 정하지만 치료하지 않는 한 얼마 안 가 사망할 것이다.
+        - |-
+          【지역소멸】
+          연극의 무대가 된 지역 그 자차체가 허무에 삼켜져 사라진다. 영향하의 모든 캐릭터(가람을 포함)는 【재봉】 난이도 10의 판정에 성공하면 탈출할 수 있다. 실패한 경우 즉시 사망한다. 엑스트라는 무조건 사망한다.
+      DTS:
+        name: 일그러짐표(바다)
+        items:
+        - |-
+          【해양오염】
+          적조가 발생한다. 혹은 해수 그 자체가 독으로 변한다. 등장한 모든 캐릭터는 【체력】 난이도 10의 판정을 해 실패할 경우 [중독] 5를 받는다. 엑스트라가 어떻게 될지는 GM이 정하지만 치료하지 않는 한 얼마 안 가 사망할 것이다.
+        - |-
+          【색채침식】
+          세계가 칠흑과 둔백으로 덧칠된다. 씬 내에 존재하는 여러 무생물과 생물이 흑백의 모노톤이 된다. 실잣는이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않아도 좋다. 
+        - |-
+          【이형의 바다】
+          바다의 생물이나 바다지기들이 이형화한다. 전쟁이 일어난다. GM은 적절하다고 생각하는 에너미를 배치할 것.
+        - |-
+          【해저화산】
+          해저화산이 분화한다. 해수는 끓어올라 주변의 생물들이 차례차례 사멸하고 바다지기들도 고통받는다. 바다의 물보라는 열탕이 된다. 구체적인 효과는 GM이 결정할 것.
+        - |-
+          【왜곡된 이정표】
+          일그러진 이정표가 내려온다. 일그러짐표（『MM』P263）를 굴릴 것.
+        - |-
+          【둔백의 바다】
+          바닷물이 둔백으로 변한다. 허무를 향한 달콤한 꼬득임으로 가득 찬 물에 닿은 자는 생각하는 것을 그만두고 의지 없는 인형이 되어 언젠가 녹아 사라질 것이다. 이 일그러짐이 1D6씬 (또는 1D6일) 이내에 사라지지 않을 경우 모든 캐릭터 (PC포함)는 엑스트라가 되어 두 번 다시 자아를 찾을 수 없다. GM이 어떠한 이유로 효과를 받지 않는 캐릭터를 설정해도 좋다.
+        - |-
+          【검은 바다】
+          바닷물이 칠흑으로 변한다. 바다 속에 있으면 한 치 앞도 보이지 않는다. [구분 : 사격]의 무기에 의한 공격의 펌블치가 +5된다.	
+        - |-
+          【벽면붕괴】
+          바다의 벽면이 무너지고 금이 간다. 금을 어떻게 하지 않는다면 수압으로 1D6씬 (혹은 1D6시간)후에 벽이 무너지고 바다가 넘쳐흐르게 된다.
+        - |-
+          【고형화】
+          바닷물이 고형화한다. 그 속에 있는 것은 얼음으로 굳힌 것 처럼 움직이지 못하고 그저 죽음을 기다린다. 엑스트라는 완전히 움직이지 못하게 된다. 에너미나 PC는 모드 [중압]&[포박]을 받는다. 이 효과는 일그러짐을 대신 받을 때 까지 매 씬, 매 라운드 개시시에 다시 적용된다.
+        - |-
+          【범람】
+          바닷물이 급격하게 늘어나 수조에서 넘쳐흐른다. 씬에 등장해있는 바다지기 클래스를 가지지 않는 캐릭터의 【HP】는 즉시 0이 된다. 단, 《대하의 인도》의 효과를 받은 캐릭터와 《수중생물》을 취득한 캐릭터는 3D6점의 【HP】를 소비하는 것으로 효과를 상쇄한다.
+        - |-
+          【소금의 눈】
+          하늘에서 눈처럼 소금이 쏟아진다. 사람들은 처음에는 기뻐하지만, 계속 쏟아지는 소금은 밭을 휩쓸고 사람들에게 해를 입힌다. 소금은 그럼에도 불구하고 멎지 않는다.
+        - |-
+          【민물화】
+          해수가 민물이 된다. 인간 중에는 기뻐하는 자들도 있지만 바다의 생물들에게는 죽음을 의미한다.
+        - |-
+          【해류소실】
+          바다의 해류가 사라진다. 바다지기의 힘을 가지고도 파도를 일으킬 수 없다. 고인 해수는 부패하고 바다의 은혜를 받아 살아가는 이들이 차례차례 병에 걸린다.
+        - |-
+          【무너진 세계】
+          바다 주변 (바다 내부가 아니라 그 외부)에서 폭발적으로 올풀림이 넓어진다. 그 규모는 GM이 결정하지만 머지않아 주민들은 죽음을 맞이할 것이다.
+        - |-
+          【술의 바다】
+          바닷물이 술이 된다. 물을 마신 모든 것은 취해 이성을 잃는다. 그 모습이 마치 암흑기에 멸망한 "나태의 고도"를 연상시킨다.	
```

The trailing space (tab) should be removed.
```suggestion
          바닷물이 술이 된다. 물을 마신 모든 것은 취해 이성을 잃는다. 그 모습이 마치 암흑기에 멸망한 "나태의 고도"를 연상시킨다.
```